src/test/java/io/vavr/collection/CollectionsTest.java

```diff
@@ -93,6 +96,41 @@ public void shouldNotBeEqualMapAndSet() throws Exception {
                 TreeMap.of(1, 2, 2, 3)), false);
     }
 
+    @Test
+    public void shouldUseMergeFunctionToHandleKeysCollisions() {
+        final Seq<Tuple2<String, Integer>> input =
+                Vector.of(Tuple.of("a",2), Tuple.of("a",55), Tuple.of("a",3), Tuple.of("b",2));
+
+        Map<String, Seq<Integer>> m = input.toMap(
+                Tuple2::_1,
+                t -> Vector.of(t._2),
+                Seq::appendAll);
+
+        assertThat(m.containsKey("a")).isTrue();
+        assertThat(m.get("a").get().size()).isEqualTo(3);
+        assertThat(m.containsKey("b")).isTrue();
+        assertThat(m.get("b").get().size()).isEqualTo(1);
+
+        Map<String, Integer> m1 = input.toMap(
+                Tuple2::_1,
+                Tuple2::_2,
+                Integer::sum);
+
+        assertThat(m1.containsKey("a")).isTrue();
+        assertThat(m1.get("a").get()).isEqualTo(60);
+        assertThat(m1.containsKey("b")).isTrue();
+        assertThat(m1.get("b").get()).isEqualTo(2);
+
+        Map<String, Integer> m2 = input.toMap(
+                Function.identity(),
+                Integer::sum);
+
+        assertThat(m2.containsKey("a")).isTrue();
+        assertThat(m2.get("a").get()).isEqualTo(60);
+        assertThat(m2.containsKey("b")).isTrue();
+        assertThat(m2.get("b").get()).isEqualTo(2);
+    }
```

Comment:
Please keep only the first test `m`.

`m2` is obsolete (see above) and `m1` does not bring anything new to the table.

Reply:
Makes sense, done.